quick_start.adoc

```diff
@@ -282,6 +283,63 @@ oc create -f <authrealm_file_name>.yaml
 At this point, any managed cluster that matches your Placement should be updated with an OAuth configuration that leverages an OpenID identity provider to connect to the proxy server that identity configuration management has stood up under the covers. Allow several minutes for the new login button to appear on your managed cluster.
 You will be able to login with the ldap user.
 
+[#openid-authentication]
+OpenID identity provider configuration provides identities for users authenticating using OpenID credentials. You can configure Red Hat Single Sign-On as an OpenID Connect identity provider.
```

Comment:
I'm struggling a bit with the wording of the first sentence. Shall we simplify to something like...
You can configure your AuthRealm to integrate with an OpenID Connect identity provider, such as Red Hat Single Sign-On.

Reply:
I have copied first sentence from openshift doc. I agree its bit hard to understand. I'll change it.